images/build/Dockerfiles/azureFunctions.JamStack.Dockerfile

```diff
@@ -4,11 +4,17 @@ ARG DEBIAN_FLAVOR
 
 ENV DEBIAN_FLAVOR=$DEBIAN_FLAVOR \
     ORYX_BUILDIMAGE_TYPE="jamstack" \
-    PATH="/usr/local/go/bin:$PATH"
+    PATH="/opt/nodejs/lts/bin:/usr/local/go/bin:$PATH"
 
 COPY --from=support-files-image-for-build /tmp/oryx/ /tmp
 RUN oryx prep --skip-detection --platforms-and-versions nodejs=12 \
     && echo "jamstack" > /opt/oryx/.imagetype \
+    && . /tmp/build/__nodeVersions.sh \
+    && /tmp/images/installPlatform.sh nodejs $NODE14_VERSION \
```

Comment:
why do we need this? jamstack image should be minimal ... node 14 is available for dynamic install anyways

Reply:
It doesn't have NPM which needed to install lerna, to have NPM it needs to install one node version on disk inside the image, so I installed node 14, but curious if there has any other ways..

Reply:
In that case i am just curious if we can solve this by using oryx prep command to install node-14 as  part of the github-action and then the user can use some sort of pre build script to install lerna .... I mean this solution of course works ... but it will add some space .. so i would confirm this with jamstack team (mitchell) if he is fine with this extra space ..

Reply:
I see, I want to try this way locally to see if it works.. if it's working then we can discuss with jamstack team with it